src/PostgREST/MediaType.hs

```diff
@@ -33,19 +35,21 @@ data MediaType
   | MTPlan MediaType MTPlanFormat [MTPlanOption]
   deriving Show
 instance Eq MediaType where
-  MTApplicationJSON == MTApplicationJSON = True
-  MTSingularJSON    == MTSingularJSON    = True
-  MTGeoJSON         == MTGeoJSON         = True
-  MTTextCSV         == MTTextCSV         = True
-  MTTextPlain       == MTTextPlain       = True
-  MTTextXML         == MTTextXML         = True
-  MTOpenAPI         == MTOpenAPI         = True
-  MTUrlEncoded      == MTUrlEncoded      = True
-  MTOctetStream     == MTOctetStream     = True
-  MTAny             == MTAny             = True
-  MTOther x         == MTOther y         = x == y
-  MTPlan{}          == MTPlan{}          = True
-  _                 == _                 = False
+  MTApplicationJSON    == MTApplicationJSON   = True
+  MTArrayJSONStrip     == MTArrayJSONStrip    = True
+  MTSingularJSONStrip  == MTSingularJSONStrip = True
+  MTSingularJSON       == MTSingularJSON      = True
+  MTGeoJSON            == MTGeoJSON           = True
+  MTTextCSV            == MTTextCSV           = True
+  MTTextPlain          == MTTextPlain         = True
+  MTTextXML            == MTTextXML           = True
+  MTOpenAPI            == MTOpenAPI           = True
+  MTUrlEncoded         == MTUrlEncoded        = True
+  MTOctetStream        == MTOctetStream       = True
+  MTAny                == MTAny               = True
+  MTOther x            == MTOther y           = x == y
+  MTPlan{}             == MTPlan{}            = True
+  _                    == _                   = False
```

Comment:
I realize this is quite ugly. I think the whole media type handling could be refactored to use https://hackage.haskell.org/package/http-media-0.8.0.0/docs/Network-HTTP-Media.html.

Then we could remove this part.

Reply:
Perhaps, maybe we should move this refactoring to a new separate issue to work on?

Reply:
@taimoorzaeem Could you open that one :pray: ?

Reply:
Yeah sure!